java/src/jmri/jmrit/operations/routes/Route.java

```diff
@@ -69,7 +67,7 @@ public void setName(String name) {
         String old = _name;
         _name = name;
         if (!old.equals(name)) {
-            setDirtyAndFirePropertyChange("nameChange", old, name); // NOI18N
+            setDirtyAndFirePropertyChange(ROUTE_NAME_CHANGED_PROPERTY, old, name); // NOI18N
```

Comment:
Just a check: The property name is changed from `namedChanged` to `routeNameChange`.
I assume you have checked the occurrences of "namedChanged" and changed them.

Reply:
Thanks Dan.  I never reference the property change string name directly.  In this case no property change previously referenced "nameChange". But the new window does pay attention to a route's name, thus the format change by me.  Easy to do a "where used" when doing it this way.